src/index.html

```diff
@@ -9,11 +9,44 @@
           maximum-scale=1.0,
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
+  <title>MOYO HEADER</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
-<body>
-
+<body class="body">
+<header class="header">
+  <strong class="logo">
```

Comment:
no need to make here such a complicated structure, just make `<a>` tag with `<img>` inside and style it accordingly

Reply:
Done.